SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+The **Falco** team and community take security bugs in all Falco projects seriously.
+
+We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
+
+## Supported Versions
+
+Security updates will typically only be applied to the latest release (at least until **Falco** reaches first stable major version).
+
+| Version.   | Supported          |
+| ---------- | ------------------ |
+| >=0.15.x   | :white_check_mark: |
```

Comment:
I don't understand why 0.15.0 here and not 0.18.0 - we don't backport security fixes to older releases at this stage.

Reply:
Right! Fixing it